src/components/checkbox/checkbox.stories.tsx

```diff
@@ -0,0 +1,110 @@
+import React, { useState } from "react";
+import { ComponentStory } from "@storybook/react";
+import { Checkbox, CheckboxGroup } from ".";
+
+export const Default: ComponentStory<typeof Checkbox> = () => {
+  const [isChecked, setIsChecked] = useState(false);
+
+  return (
+    <Checkbox
+      label="Example checkbox"
+      name="checkbox-default"
+      checked={isChecked}
+      onChange={(e) => setIsChecked(e.target.checked)}
+    />
+  );
+};
+
+export const Sizes: ComponentStory<typeof Checkbox> = () => (
+  <>
+    <Checkbox
+      label="Small"
+      key="checkbox-small"
+      name="checkbox-small"
+      size="small"
+    />
+    <Checkbox
+      label="Large"
+      key="checkbox-large"
+      name="checkbox-large"
+      size="large"
+    />
+  </>
+);
+
+export const Disabled: ComponentStory<typeof Checkbox> = () => (
```

Comment:
Same as in `checkbox-validations.stories.tsx` file you could use a template for that